### PR TITLE
Allow passing -j to compileOF.sh for parallel compilation

### PR DESCRIPTION
**Pull request by flvi0**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ PLATFORM SPECIFIC
 
 ### windows
 	+ activated high precission timming to fix inaccurate fps calculations
+	+ allow passing -j<num_jobs> to compileOF.sh
```
Comment (pizthewiz) [inline CHANGELOG.md]: Is this change for Windows only?

**Comment by pizthewiz** [comment]
Seems like an uncontroversial change given the discussion in #3970.

File: CHANGELOG.md (hunk 1 above)
Comment (flvi0) [inline CHANGELOG.md]: Ooops! Thanks for spotting, actually it's Linux only, my bad, early morning, first commit to oF ;)
Will fix, force-push and recreate pull request!

Comment (pizthewiz) [inline CHANGELOG.md]: No worries, force-push should do the trick if you didn't want another commit, pretty sure that's how [rebasing a PR](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) is typically done too.

**Comment by flvi0** [comment]
Done. Sorry :)

**Comment by pizthewiz** [comment]
No worries at all, thanks for the PR! 

@arturoc, this PR has the simplified jobs assignment you mentioned in #3970.

**Comment by flvi0** [comment]
Haha I wonder why it takes so long...

    ...
    $ git clone --depth=10 https://github.com/openframeworks/openFrameworks.git openframeworks/openFrameworks
    Cloning into 'openframeworks/openFrameworks'...
    remote: Counting objects: 7459, done.

    Broadcast message from root@testing-gce-16cc3342-0073-4bbc-90d7-25222a49af01
	(unknown) at 7:33 ...
    The system is going down for power off NOW!

You might want to manually stop/restart this.

**Comment by bilderbuchi** [comment]
> You might want to manually stop/restart this.

I just did.

**Comment by arturoc** [comment]
thanks!

**Pull request merged**